src/python/WMCore/MicroService/MSUnmerged/MSUnmerged.py

```diff
@@ -771,24 +771,25 @@ def uploadRSEToMongoDB(self, rse, fullRSEToDB=False, overwrite=True):
             raise MSUnmergedPlineExit(msg)
         return rse
 
-    def getStatsFromMongoDB(self, **kwargs):
+    def getStatsFromMongoDB(self, detail=False, **kwargs):
         """
         Auxiliary method to serve the APIs of the REST calls for the service.
         Implements various queries to MongoDB based on the parameters passed.
         :param rse=rseName:      String representing the RSE name to query the database for
-        :param rse_type=rseType: String representing the RSE type to query the database for
         :return:                 Dictionary with the type of query + the data returned from the database
         """
         data = {}
         if kwargs.get('rse'):
-            data["query"] = 'rse=%s' % kwargs['rse']
+            data["query"] = 'rse=%s&detail=%s' % (kwargs['rse'], detail)
             mongoProjection = {"_id": False,
                                "name": True,
                                "pfnPrefix": True,
                                "isClean": True,
                                "timestamps": True,
-                               "counters": True,
```

Comment:
shouldn't we explicitly set the projection to `False` if no details are required? Maybe could be accomplished through:
```
....
"dirs": detail,
...
```

Reply:
Sorry @amaltaro  this is not how MongoDB works unfortunately. You can imagine my dissapointment when I found out that the only field of the document allowd to be set as `False` in the projection is `_id` otherwise you get the following exception [1] once you try to use the projection.  And speaking of which there is one more place where this needs to be fixed [2]. Which was actually the reason to not being able to use the mongoProjection to its full power when I was testing it back then. 

[2]
https://github.com/dmwm/WMCore/blob/39e1ecdfd581c497aed0d00de873d66d7ee448e4/src/python/WMCore/MicroService/MSUnmerged/MSUnmergedRSE.py#L80


[1]
```
In [87]: mongoProjection
Out[87]: 
{'_id': False,
 'name': True,
 'pfnPrefix': True,
 'isClean': True,
 'timestamps': True,
 'counters': True,
 'dirs': False}

In [88]: msUnmergedColl.find_one({'name': rse['name']}, mongoProjection)
...
OperationFailure: Projection cannot have a mix of inclusion and exclusion.
```